sound/soc/sof/ipc4-topology.c

```diff
@@ -2012,6 +2028,7 @@ sof_ipc4_process_set_pin_formats(struct snd_sof_widget *swidget, int pin_type)
 
 			if (pin_format_item->pin_index == i - pin_format_offset) {
 				*pin_format = *pin_format_item;
+				pin_format->buffer_size *= process->buffer_period_size;
```

Comment:
naming is awful. suggest  "buffer_size_multiplier". It doesn't help to refer to the period and is't not a buffer or period_size in the first place.

Reply:
Ack, will update.